src/Xamarin.Android.Build.Tasks/Tasks/ResolveAndroidTooling.cs

```diff
@@ -78,16 +78,9 @@ public override bool RunTask ()
 			string toolsZipAlignPath = Path.Combine (AndroidSdkPath, "tools", ZipAlign);
 			bool findZipAlign = (string.IsNullOrEmpty (ZipAlignPath) || !Directory.Exists (ZipAlignPath)) && !File.Exists (toolsZipAlignPath);
 
-			var commandLineToolsDir = MonoAndroidHelper.AndroidSdk.GetCommandLineToolsPaths (CommandLineToolsVersion)
-				.FirstOrDefault () ?? "";
-
-			var lintPaths = new string [] {
-				LintToolPath ?? string.Empty,
-				commandLineToolsDir,
-				Path.Combine (commandLineToolsDir, "bin"),
-				Path.Combine (AndroidSdkPath, "tools"),
-				Path.Combine (AndroidSdkPath, "tools", "bin"),
-			};
+			var lintPaths = MonoAndroidHelper.AndroidSdk.GetCommandLineToolsPaths (CommandLineToolsVersion)
+				.Select (p => Path.Combine (p, "bin"))
+				.ToArray ();
```

Comment:
I think you could leave out the `.ToArray()`, because we just use `foreach` over `lintPaths` once -- probably don't need to create the array?